extension/lib/squire-raw.js

```diff
@@ -1434,7 +1438,9 @@ var handleEnter = function ( self, shiftKey, range ) {
     // Remove any zws so we don't think there's content in an empty
     // block.
     self._recordUndoState( range );
-    addLinks( range.startContainer, root, self );
+    if (self._config.richText) {
```

Comment:
Squire will still respond to spacebar, so we'd need to do something similar in `space`. 

But preferably we'd do it in a form of a PR to Squire directly: https://github.com/neilj/Squire/issues/375

Reply:
Yup, sounds like a proper solution :+1:

Reply:
I went ahead and forked it at https://github.com/FlowCrypt/Squire - you can make a PR there, then use the modified Squire build here in this PR. Later you can make PR from the new fork to their original repo, too.

Reply:
Great!

> ERROR: Permission to FlowCrypt/Squire.git denied to limonte.

Please add me as a collaborator to that repo, thanks!

Reply:
https://github.com/FlowCrypt/Squire/invitations done

Reply:
@tomholub I updated the `squire-raw.js` accordingly to `FlowCrypt/Squire`, the PR is ready for review.